apps/src/blockly/customBlocks/googleBlockly/commonBlocks.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Defines blocks useful in multiple blockly apps
+ */
+
+export const blocks = {
+  installJoinBlock(blockly) {
+    blockly.Blocks.text_join_simple = blockly.Blocks.text_join;
```

Comment:
for my information, where are `blockly.Blocks.text_join` and `blockly.JavaScript.text_join` defined?

Reply:
Included with core Blockly